docs/sources/release-notes/upgrading-helm-from-2.1-to-3.0.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "TODO: Upgrading to mimir-distributed 3.0"
+menuTitle: "TODO: Helm chart v3.0"
+description: "Upgrade guide for the Grafana Mimir Helm chart version 3.0"
+weight: 100
+---
+
+# Grafana Mimir Helm chart version 3.0 Upgrade Guide
+
+## Breaking Changes
+
+Several parameters available in version 2.1 of the mimir-distributed Helm chart have changed.
+Follow these steps to migrate your Helm values file to 3.0.
+
+- **Configuration Storage has changed**
+    - Mimir configuration is now stored in a ConfigMap by default instead of a Secret
```

Comment:
```suggestion
    - Mimir configuration is now stored in a ConfigMap by default instead of a Secret
    - In case of providing the configuration in a user managed Secret (**`useExternalConfig=true`**), you must set `configStorageType=Secret`. Conversely, it is now possible to use a ConfigMap to manage your configuration.
```